README.md

```diff
@@ -137,6 +137,14 @@ HttpRequest("long.source.of/data")
   .stream()
 ```
 
+### Cookies
+
+Servers may use cookies to authenticate users or exchange other information. To make sure that the browser
+adds a previously obtained cookie to subsequent requests of the client, `withCredentials` is offered:
```

Comment:
This is not entirely true. `withCredentials` only affects cross-domain requests. As stated on [MDN](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/withCredentials), "Setting withCredentials has no effect on same-site requests."